seeds/keep3r_network/ethereum/keep3r_network_ethereum_view_job_log_postgres.csv

```diff
@@ -0,0 +1,4 @@
+tx_hash,amount,keeper,token
```

Comment:
can we remove the word ```postgres``` in the seed file name? we will want to move away from that standard